regex.rst

```diff
@@ -1,2 +1,14 @@
 Regex
```

Comment:
@vanessa-andrews can you add Regex.Word which is a alias for the get word regular expression